README.md

```diff
@@ -0,0 +1,26 @@
+# NearNodeFlash Pages
+
+# Create mkdocs Environment
+
+```bash
+$ git@github.com:NearNodeFlash/NearNodeFlash.github.io.git
+$ cd NearNodeFlash.github.io.git
+$ python3 -m venv venv
+$ . venv/bin/activate
+(venv) $ pip install -r requirements.txt
+```
+
+# Run mkdocs Server
+
+```bash
+(venv) $ mkdocs serve
+INFO     -  Building documentation...
+[...]
+INFO     -  Documentation built in 0.22 seconds
+INFO     -  [10:59:28] Watching paths for changes: 'docs', 'mkdocs.yml'
+INFO     -  [10:59:28] Serving on http://127.0.0.1:8000/
+```
+
+Now point your browser at http://127.0.0.1:8000/
```

Comment:
Surround the link with angle brackets...<http://127.0.0.1:8000>

Reply:
I think this needs to be formatted differently to avoid confusion (as it did for Tony). Maybe have an opening preamble that says "To run mkdocs server locally, execute `mkdocs serve`. The output will appear similar to below, with the localhost URL listed at the end"

Reply:
done